### DISCUSSION TITLE
Predictable port names in exported verilog

### DISCUSSION THREAD

**Discussion by Linseed** [FPGAwars/icestudio#846] [Ideas]
This is very useful software which I would like to use on another FPGA board via the exported verilog file. However, the port names in the verilog file are random and it is hard to follow which one is which. It would be good if the export routine would use assigned the port names in the graphical layout.

Thank you very much!

**Comment by Jesus89** [comment]
Related to https://github.com/FPGAwars/icestudio/issues/226

**Comment by cavearr** [comment]
Hello! I am reorganizing the issues, trying to give more visibility and be able to better organize and prioritize the work and my roadmap (I will publish it very soon), I am converting some issues that are **new feature requests** as **ideas** on Github **discussions**.

Thanks for understanding.
